binding/tfjs_backend.cc

```diff
@@ -294,7 +299,10 @@ void CopyTFE_TensorHandleDataToStringArray(napi_env env,
   const size_t expected_tensor_size =
       (limit - static_cast<const char *>(tensor_data));
   if (expected_tensor_size - byte_length) {
```

Comment:
I'd suggest changing from `if (expected_tensor_size - byte_length)` to `if (expected_tensor_size != byte_length)`